var/spack/repos/builtin/packages/sourmash/package.py

```diff
@@ -44,5 +50,3 @@ def install(self, spec, prefix):
             join_path("target", "release", "libsourmash.{}".format(lib_ext)),
             join_path(site_packages, "_lowlevel__lib.so"),
         )
-        # patch invalid read mode
-        filter_file(r"'(.*)'\), 130\)", r"'\1'))", join_path(site_packages, "_lowlevel.py"))
```

Comment:
What's the reason for this change? Is it no longer needed by any version or only by 4.8.4?

Reply:
As the author of the original spec, that filter was necessary for me to get the last version deployed on our Rocky8 Icelake cluster ... but, at the time, I was using a `%gcc@13` stack. That was proving endlessly painful and I've since scrapped the whole lot and rebuilt everything with `@12`.

Pulling the PR, can confirm that this line isn't needed for either version with my current stack, so I assume this was related to `gcc@13` itself and/or to a difference in the dependencies pulled in by that stack in comparison to that of `@12`. Both versions build and run a test through.

Reply:
I hit the same error when packaging sourmash for [nixpkgs](https://github.com/NixOS/nixpkgs/pull/230596#issuecomment-1608537072), and it also went away when upgrading to `maturin>=1`.

Reply:
Yeah, that's the CFFI error I was getting back then as well. Took a while to track it down!